packages/eslint-plugin/src/configs/react-legacy.js

```diff
@@ -10,4 +10,13 @@ module.exports = {
     'jsdoc/check-tag-names': 'off',
     '@griffel/no-shorthands': 'off',
   },
+  overrides: [
```

Comment:
this should go to `configs/base-legacy.js` as it has nothing specific with react